code/__DEFINES/components.dm

```diff
@@ -150,6 +150,7 @@
 #define COMSIG_MOB_HUD_CREATED "mob_hud_created"				//from base of mob/create_mob_hud(): ()
 #define COMSIG_MOB_ATTACK_HAND "mob_attack_hand"				//from base of
 #define COMSIG_MOB_ITEM_ATTACK "mob_item_attack"				//from base of /obj/item/attack(): (mob/M, mob/user)
+	#define COMPONENT_ITEM_NO_ATTACK 1

```

Comment:
why the tab in front?

Reply:
For readability. COMPONENT_ITEM_NO_ATTACK is basically a possible return value of COMSIG_MOB_ITEM_ATTACK. This is the standard way of writing this.

Reply:
Sounds fair.